.travis.yml

```diff
@@ -14,9 +14,11 @@ env:
     - USE_DEB=true
```

Comment:
If you want to use `UPSTREAM_WORKSPACE=file`, then `USE_DEB` needs to be either `false` or simple not defined.